Octokit/Models/Response/ActivityPayloads/PushWebhookCommitter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents the author or committer to a Git commit. This is the information stored in Git and should not be
+    /// confused with GitHub account information.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PushWebhookCommitter : Committer
```

Comment:
The fields in the `PushWebhookCommitter` are not a superset of `Committer` so we shouldn't use inheritance here.  We should just make this response model contain the fields that the webhook payload will have:

``` json
      "committer": {
        "name": "baxterthehacker",
        "email": "baxterthehacker@users.noreply.github.com",
        "username": "baxterthehacker"
      },
```

Reply:
Resolved in 4905d42e85b9b91c0712c5a0b5afe29757975aa7